package.json

```diff
@@ -71,6 +71,7 @@
     "start": "npm run build && npm run serve",
     "translate": "node dist/lib/checkTranslations.js",
     "rebuild": "npm run clean && npm run build",
-    "release": "npm run rebuild && npm run lint && npm run format"
+    "release": "npm run rebuild && npm run lint && npm run format",
+    "test": "jest"
```

Comment:
Not really necessary, one can just run `yarn jest`.

Reply:
Got it, I'll remove this.  I was just thinking I would add it as a convenience thing since for most projects I've seen the standard has been "run `npm/yarn test` to run tests"